src/App/Entity/Talk.php

```diff
@@ -77,57 +77,58 @@ public static function fromStandardTalk(
         DateTimeImmutable $time,
         Speaker $speaker,
         string $title,
-        string $abstract
+        string $abstract,
```

Comment:
In the form I would simply say not empty is sufficient. But I don't see why you would set the value to null in the DB if you get an empty string?